L1Trigger/TrackTrigger/python/TrackQualityParams_cfi.py

```diff
@@ -1,8 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 TrackQualityParams = cms.PSet(qualityAlgorithm = cms.string("GBDT_cpp"), #None, Cut, NN, GBDT, GBDT_cpp
-                              ONNXmodel = cms.FileInPath("L1Trigger/TrackTrigger/data/clf_GBDT_emulation_newKF_digitized.json"),
-                              #ONNXmodel = cms.FileInPath("L1Trigger/TrackTrigger/data/gbdt.json"),
+                              ONNXmodel = cms.FileInPath("L1Trigger/TrackTrigger/data/L1_TrackQuality_GBDT_emulation_digitized.json"),
```

Comment:
Please add a comment saying this is for use with the HYBRID_NEWKF algorithm. 
Can you remind me what happens for people running the old HYBRID chain? Will they TrackQuality with the same JSON?

Reply:
I have added the comment, those running the old HYBRID chain will still use this JSON as it has similar performance in both the HYBRID_NEWKF and HYBRID chains